<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #AACCCCCC</string>
  </property>
  <widget class="QLabel" name="icon_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="message_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>811</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string>message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="content_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>961</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="esc_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>Frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>esc_button</sender>
   <signal>clicked()</signal>
   <receiver>Frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
